webviz_subsurface/plugins/_co2_leakage/_utilities/callbacks.py

```diff
@@ -310,6 +310,10 @@ def create_map_layers(
         well_data = dict(well_pick_provider.get_geojson(selected_wells, formation))
         if "features" in well_data and len(well_data["features"]) == 0:
             warnings.warn(f'Formation name "{formation}" not found in well picks file.')
+        else:
```

Comment:
Might not need to put "else" here, but I guess it makes no difference. Without "else" the for loop below will not run since len(...) = 0.

Reply:
Fixed